app/views/layouts/application.html.haml

```diff
@@ -33,3 +36,33 @@
     .container
       = render "layouts/shared/notification"
       = yield
+      -if current_user.present? && current_user.role.to_s == "Admin"
+        = form_for @company, remote: true do |f|
```

Comment:
This form should be available to all the pages, as it is a link in the navigation bar, so this form is in application layout.